src/index.js

```diff
@@ -18,11 +18,7 @@ process.on('unhandledRejection', reason => {
 let graphQLServer;
 const port = +process.env.PORT || 3090;
 const load = async props => {
-  if (!graphQLServer) {
-    graphQLServer = new GraphQLServer(props);
-  } else {
-    await graphQLServer.reload(() => props);
```

Comment:
`graphQLServer.reload` is a method created by the forked version of graphql-yoga.